src/main/resources/assets/mythicmetals/lang/de_de.json

```diff
@@ -0,0 +1,1146 @@
+  "advancement.mythicmetals.mythril_drill.title": "Graben und Graben macht uns frei",
+  "advancement.mythicmetals.orichalcum.description": "Finde Orichalcum, ein Hallowederz verstreut in den Deepslate Höhlen",
+  "advancement.mythicmetals.orichalcum.title": "Orkische Stärke",
+  "advancement.mythicmetals.orichalcum_hammer.description": "Erstelle einen Orichalcum Hammer, mit dem du schneller lange Tunnel graben kannst.",
+  "advancement.mythicmetals.orichalcum_hammer.title": "Du weißt, wie spät es ist",
+  "advancement.mythicmetals.osmium.description": "Finde Osmium hoch oben in den Bergen",
+  "advancement.mythicmetals.osmium.title": "Wann kann ich ein Windrad bauen?",
+  "advancement.mythicmetals.palladium.description": "Finde Palladium, normalerweise unter dem Lava-Ozean im Nether",
+  "advancement.mythicmetals.palladium.title": "Heißeste Baked Beans",
+  "advancement.mythicmetals.prometheum.description": "Finde Prometheum tief in Dschungeln oder Lush Caves",
+  "advancement.mythicmetals.prometheum.title": "Üppiges Gemüse",
+  "advancement.mythicmetals.runite.description": "Finde Runite Ore, knapp unter der Oberfläche, versteckt in den Wänden",
+  "advancement.mythicmetals.runite.title": "Gerade zur rechten Zeit geboren...",
+  "advancement.mythicmetals.star_platinum.description": "Schmiede Stern Platin aus Sternrit and Platin",
+  "advancement.mythicmetals.star_platinum.title": "Das bizarrste Metall der Welt",
+  "advancement.mythicmetals.starrite.description": "Finde Sternrit, ein reichlich vorhandener Kristall, der gewöhnlich im Ende gefunden wird",
+  "advancement.mythicmetals.starrite.title": "Gefallene Sternentropfen",
+  "advancement.mythicmetals.steel.description": "Schmiede Stahl, eine einfache Legierung aus Eisen and Manganese",
+  "advancement.mythicmetals.steel.title": "Aber Stahl ist schwerer als...",
+  "advancement.mythicmetals.stormyx.description": "Finde Stormyx tief im Nether versteckt",
+  "advancement.mythicmetals.stormyx.title": "Magischee Metalle",
+  "advancement.mythicmetals.stormyx_shell.description": "Bekomme eine Stormyx Shell, selten aus Stormyx Erzen",
+  "advancement.mythicmetals.stormyx_shell.title": "Der Nether Diamand",
+  "advancement.mythicmetals.stormyx_shield.description": "Crafte einen Stormyx Shild aus einer Stormyx Shell",
+  "advancement.mythicmetals.stormyx_shield.title": "Gebet für besseren Schutz",
+  "advancement.mythicmetals.unobtainium_template.description": "Finde ein Unerhaltbarium Schmieden Template in einer ancient city, die es Ihnen ermöglicht, mächtige Ausrüstung aus Unerhaltbarium-Legierungen zu schmieden",
+  "advancement.mythicmetals.unobtainium_template.title": "Das Geheimnis des Sculk",
+  "advancement.mythicmetals.unobtainium.description": "Bekomme Unerhaltbarium, für gewöhnlich Tief im Untergrund zu finden",
+  "advancement.mythicmetals.unobtainium.title": "Wie hast du das bekommen?",
+  "advancement.mythicmetals.superhot.title": "Ich bin on Fire!",
```

Comment:
This seems partly untranslated, is this correct?

Reply:
That is correct. Translating (to be) on fire to German would be possible, but doesn't really make sense in that context, as this idiom doesn't exist in German.